### DISCUSSION TITLE
macOS runners in pending state not executing jobs

### DISCUSSION THREAD

**Discussion by buffer** [community/community#82805] [Actions]
### Select Topic Area

Bug

### Body

During the last days I have observed that actions which are supposed to be run on macOS runners are in pending state (Waiting for pending jobs) and time out after 24 hours. This is happening just for macOS runners so it seems to me something is not working properly for such runners.

**Comment by JesseTG** [comment]
This occurs for me as well. [An open-source project I'm working on](https://github.com/JesseTG/melonds-ds) uses GitHub Actions extensively to produce builds. Generally, these builds only take a few minutes each. However, the macOS runners _sometimes_ hang and stall for hours.

Here are some examples from yesterday alone:

- https://github.com/JesseTG/melonds-ds/actions/runs/7468340347/job/20323587446
- https://github.com/JesseTG/melonds-ds/actions/runs/7468541145/job/20324180760
- https://github.com/JesseTG/melonds-ds/actions/runs/7468340347/job/20323588307
